models/marts/customer_metrics.sql

```diff
@@ -0,0 +1,13 @@
+{{
```

Comment:
Is this model just to show how to use variables in the query and configuration? And when do you usually teach this?

Reply:
Yeah this was a random fun with variables thing I did. I can remove this.